### DISCUSSION TITLE
[Feature] allow user to manually tag people on picture

### DISCUSSION THREAD

**Discussion by soubiliz** [immich-app/immich#4835] [Feature Request]
### The feature

Face détection will always be imperfect, sometime the face isn't detected because it's to close to another face, because it's partially hidden, etc.

But those picture still contain someone and when looking for memories containing that unrecognized face can't correct the system.

I would like to suggest a way to manually tag an area as someone's face, it doesn't have to be used as data for the model but just as a tag.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486
